src/aap_eda/wsapi/consumers.py

```
@@ -100,6 +102,204 @@ class Event(Enum):
+            raise InvalidTokenError("No authorization header found")
+
+        parts = auth_header.split()
+        if len(parts) != 2 or parts[0] != "Bearer":
+            raise InvalidTokenError("Invalid authorization header format")
+
+        token = parts[1]
+        self._token_payload = parse_jwt_token(token)
+
+        # Security: Reject refresh tokens - only access tokens allowed
+        token_type = self._token_payload.get("token_type")
+        if token_type != "access":
```

Ruff (0.15.20)

[error] Possible hardcoded password assigned to: "token_type"

(S105)